src/test/xspec/dtbook-to-epub3.xspec

```diff
@@ -820,13 +820,34 @@
     <!-- implicit epub:type tested in element test, otherwise nothing interesting to test -->
 
     <!-- epigraph -->
+    <!-- The nordic guidelines do not allow for epigraph. So the
+         following is for completeness sake and takes some inspiration
+         from https://www.w3.org/TR/dpub-aria-1.0/#doc-epigraph -->
     <x:scenario label="when processing a epigraph element">
-        <x:context>
-            <dtbook:epigraph/>
-        </x:context>
-        <x:expect label="the resulting element should be a p with an added epub:type of epigraph">
-            <html:p id="..." epub:type="epigraph" class="epigraph"/>
-        </x:expect>
+	<x:scenario label="on its own">
+            <x:context>
+		<dtbook:epigraph/>
+            </x:context>
+            <x:expect label="the resulting element should be a blockquote with an added epub:type of epigraph">
+		<html:blockquote id="..." epub:type="epigraph" class="epigraph"/>
```

Comment:
`nordic-html5.rng` defines epigraph as a `<p>`. The RelaxNG and Schematron should be updated where necessary.

Reply:
...also, in the RelaxNG/Schematron, maybe we should still allow `<p>` as epigraph in addition to `<blockquote>` so that existing books that are valid don't become invalid all of a sudden.

Reply:
So I changed the rng so that the epigraph should now use the element `blockquote`. I also changed the content model to flow as that seemed more appropriate.

I found no place in the schematron that I thought needed changing.

As far as backwards compatibility and also allowing `p`. I think this might be tricky especially since I'm also changing the content model. Since nobody was using `dtbook:epigraph` anyway I think we are on the safe side. And btw existing books were already invalid since they contain nested `html:p`.

Reply:
I'm not sure what to do about the comment about the HTML content model (see [nordic-html5.rng#L1791](https://github.com/nlbdev/nordic-epub3-dtbook-migrator/blob/master/src/main/resources/xml/schema/nordic-html5.rng#L1791)). What is that refering to? To `html:p`? So this would have to be changed to the content model of `html:blockquote`. Where did you get that information?